ci-operator/config/openshift/release/openshift-release-master__stable-4.12.yaml

```diff
@@ -0,0 +1,261 @@
+base_images:
```

Comment:
We don't have anything that looks like the `openshift-release-master__stable-4.12.yaml` pattern to me.  The only existing `__stable-4.*` are 4.8 and older update jobs:

```console
$ ls ci-operator/config/openshift/release/openshift-release-master__stable-4.*.yaml
...
ci-operator/config/openshift/release/openshift-release-master__stable-4.8-upgrade-from-stable-4.4.yaml
ci-operator/config/openshift/release/openshift-release-master__stable-4.8-upgrade-from-stable-4.5.yaml
```

and the other z-rollback jobs all use `__nightly`:

```console
$ git --no-pager grep oldest-supported-z-rollback: -- ci-operator/config/openshift/release
ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.15.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:  oldest-supported-z-rollback:
ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml:  oldest-supported-z-rollback:
```

Can we use `openshift-release-master__nightly-4.12.yaml` here too, to match that pattern?

Reply:
QE is used to separate tests on nightly builds and stable builds into `__nightly` and `__stable` files respectively. So we can easily identify whether nightly build or stable build we are using for the test from the job name, like: 
periodic-ci-openshift-release-master-**stable**-4.12-gcp-ipi-ovn-ipsec-upgrade-rollback-oldest-supported, 
periodic-ci-openshift-release-master-**nightly**-4.14-e2e-aws-ovn-upgrade-rollback-oldest-supported.
Dev might use a different rule. If you prefer to merge the stable test into `__nightly` file, It's fine with me too.